seedlet/assets/sass/blocks/utilities/_style.scss

```diff
@@ -14,22 +14,33 @@
 .alignleft {
 	/*rtl:ignore*/
 	text-align: left;
-	/*rtl:ignore*/
-	float: left;
 	margin-top: 0;
-	/*rtl:ignore*/
-	margin-right: var(--global--spacing-horizontal);
-	margin-bottom: var(--global--spacing-vertical);
 }
 
 // Targeting the .entry-content class is necessary to ensure these styles
 // only apply when the block isn't nested.
 .entry-content > .alignleft {
 	max-width: var(--responsive--aligndefault-width);
-	@extend %responsive-alignleft-mobile;
 	@extend %responsive-alignleft;
 }
 
+@include media(mobile) {
+
+	.alignleft {
+
+		/*rtl:ignore*/
+		float: left;
+
+		/*rtl:ignore*/
+		margin-right: var(--global--spacing-horizontal);
+		margin-bottom: var(--global--spacing-vertical);
+	}
+
+	.entry-content > .alignleft {
+		max-width: calc(50% - var(--responsive--alignleft-margin));
```

Comment:
We had a convo about this in Slack, but wanted to post it here for good measure: 

What do you think about renaming this variable to `--responsive--alignleft-width`, and removing the unused `responsive-alignleft/right` mixins?

Can either be in this PR or a follow up.

Reply:
yeah, I'm going to work on this and maybe afterwards we can PR TT1 with the same changes